lib/standards/html-elms.js

```diff
@@ -822,7 +822,7 @@ const htmlElms = {
   },
   svg: {
     contentTypes: ['embedded', 'phrasing', 'flow'],
-    allowedRoles: ['application', 'document', 'img'],
+    allowedRoles: true,
```

Comment:
Is this the only change we need to make for an updated ARIA in HTML?

Reply:
That I'm not sure about, I just saw the ticket for svgs in particular being updated.